Description: update to arrayvec 0.7
 Upstream no longer depends on arrayvec so I had to patch this myself
Author: Peter Michael Green <plugwash@debian.org>

--- rust-gimli-0.19.0.orig/Cargo.toml
+++ rust-gimli-0.19.0/Cargo.toml
@@ -24,7 +24,7 @@ categories = ["development-tools::debugg
 license = "Apache-2.0/MIT"
 repository = "https://github.com/gimli-rs/gimli"
 [dependencies.arrayvec]
-version = "0.4.6"
+version = "0.7"
 default-features = false
 
 [dependencies.byteorder]
--- rust-gimli-0.19.0.orig/src/read/cfi.rs
+++ rust-gimli-0.19.0/src/read/cfi.rs
@@ -1823,7 +1823,7 @@ impl<R: Reader> UninitializedUnwindConte
 }
 
 const MAX_UNWIND_STACK_DEPTH: usize = 4;
-type UnwindContextStack<R> = ArrayVec<[UnwindTableRow<R>; MAX_UNWIND_STACK_DEPTH]>;
+type UnwindContextStack<R> = ArrayVec<UnwindTableRow<R>, MAX_UNWIND_STACK_DEPTH>;
 
 /// An unwinding context.
 #[derive(Clone, Debug, PartialEq, Eq)]
@@ -2288,7 +2288,7 @@ impl<'a, R: Reader> UnwindTable<'a, R> {
 // comparisons that way.
 #[derive(Clone, Debug)]
 struct RegisterRuleMap<R: Reader> {
-    rules: ArrayVec<[(Register, RegisterRule<R>); 32]>,
+    rules: ArrayVec<(Register, RegisterRule<R>), 32>,
 }
 
 impl<R: Reader> Default for RegisterRuleMap<R> {
